### PR TITLE
Feature/hunter

### PR DESCRIPTION
**Pull request by zmf963**
Add Hunter data source.

### CODE REVIEW

File: uncover/agent/hunter/request.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package hunter
+
+type Request struct {
+	USERNAME   string `json:"username"`
```
Comment (forgedhallpass) [inline uncover/agent/hunter/request.go:4]: Why are the `USERNAME` and `APIKEY` fields all uppercase?

Comment (zmf963) [reply uncover/agent/hunter/request.go]: Sorry, this is a mistake of mine.

**Comment by zmf963** [comment]
> @zmf963感谢您添加更多来源。 你能告诉我们如何注册这个平台来获取 API 密钥吗？

You need to register the platform account first, and then log in to the personal center page, where you can see API-KEY

https://user.skyeye.qianxin.com/user/register?next=https%3A//hunter.qianxin.com/api/uLogin&fromLogin=1
图片

图片

**Comment by zmf963** [comment]
This is the home page address of the hunter data source   `https://hunter.qianxin.com/`
This is the address of the relevant API documentation `https://hunter.qianxin.com/home/helpCenter?r=5-1-2`

**Comment by zmf963** [comment]
This is the effect of running

图片

This is to configure the hunter API-KEY
```sh
root:uncover/ (Feature/hunter) # cat /root/.config/uncover/provider-config.yaml                                         [14:52:07]
shodan: []
censys: []
fofa: []
quake: []
hunter:
  - 5eca15f7ca7bc5f753023a35728947fbfd136daefe3452e300b1xxxxxxxxxxxx
```

**Review by ehsandeep — APPROVED** [review]

**Review by ehsandeep — APPROVED** [review]
```console
echo Jira | ./uncover -e hunter -silent

34.159.58.69:443
34.159.132.250:443
23.64.223.249:443
78.104.35.162:443
37.152.187.18:8080
18.66.137.107:80
45.60.171.100:8418
137.117.134.124:443
212.174.76.180:443
35.156.78.142:80
```

**Review by Mzack9999 — APPROVED** [review]

**Pull request merged**